src/main/java/test/testvolodya/CreateAddress.java

```diff
@@ -9,33 +25,42 @@
 import main.java.test.test;
 import org.testng.annotations.Listeners;
 import org.testng.annotations.Test;
-
 import static org.testng.Assert.assertEquals;
 
+// Listeners are waiting for the error and if it occurs - make a screenshot of error page
 @Listeners(test.class)
 
+// CreateAddress class
 public class CreateAddress extends test {
 
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Verify that new address entry can be created")
     @Test
     public void createAddress() {
-        // Log In
-        new LogInHelper();
+        // Log In to PrestaShop
+        UserPage userPage = new LogInHelper().getUserPage();
 
         // Change language to English
-        new CertainLanguage().clickLanguage().clickEnglishInDropDown();
+        CertainLanguage certainLanguage = new CertainLanguage().clickLanguage().clickEnglishInDropDown();
 
-        // Click addresses card-link
+        // Click 'Add new address' card-link
         new UserPage().getUserPageContainer().clickAddresses();
 
-        // Verify that we are on the page we expected (Create new address)
-        String actualPageName = new AddressesPage().getActualPageName().getText();
-        String expectedPageName = "New address";
-
         // Clear all the fields before filling in
-        new NewAddress().clearField("alias", "firstName", "lastName", "company", "vatNumber", "myAddress",
-                "addressComplement", "zipPostalCode", "city", "country", "phone");
+        new NewAddress().clearField(
+                "alias",
+                "firstName",
+                "lastName",
+                "company",
+                "vatNumber",
+                "myAddress",
+                "addressComplement",
+                "zipPostalCode",
+                "city",
+                "country",
+                "phone");
 
-        // Fill in all required fields from addresses repository from create address1.
+        // Fill in all necessary fields from addresses repository.
         Address address = new AddressRepository().getAddress1();
         new NewAddress().fillInField(address);
 
```

Comment:
try and move it from test

Reply:
Something like this:
        // Fill in all necessary fields from addresses repository.
        Address address = new AddressRepository().getAddress1();
        NewAddress newAddress = new NewAddress().fillInField(address);
? :)

Reply:
I can't move it from test.
I need it for making this test(